src/test/scala/is/hail/utils/RichIndexedRowMatrixSuite.scala

```diff
@@ -13,12 +13,14 @@ import org.testng.annotations.Test
 class RichIndexedRowMatrixSuite extends SparkSuite {
 
   @Test def testToBlockMatrixDense() {
-    val m = 4
-    val n = 3
+    val rows = 6L
+    val cols = 3L
```

Comment:
Given the complexity of the conversion depending on excessCols, etc, I think we should have better coverage of edge cases in the tests. I'd make make rows = 9 and cols = 6 and use block sizes: 1, 2, 3, 4, 6, 7, 9, 10. This gives all combinations of excess / noExcess in rows and in columns and has all cases of non-trivially dividing or exactly equaling the number of rows or columns.